firmware/targets/f7/furi_hal/furi_hal_info.c

```diff
@@ -298,15 +298,62 @@ void furi_hal_info_get(PropertyValueCallback out, char sep, void* context) {
         property_value_out(&property_context, NULL, 2, "radio", "alive", "false");
     }
 
+    // RTC flags
     property_value_out(
         &property_context,
         "%u",
         2,
         "system",
         "debug",
         furi_hal_rtc_is_flag_set(FuriHalRtcFlagDebug));
+    property_value_out(
+        &property_context, "%u", 2, "system", "lock", furi_hal_rtc_is_flag_set(FuriHalRtcFlagLock));
+    property_value_out(
+        &property_context,
+        "%u",
+        3,
+        "system",
+        "hand",
```

Comment:
Here goes difficult part ;-) those values are meant to form hierarchy usable for filtering. Simply using flag names is not good.